components/scripts/lib/maven.sh

```diff
@@ -40,8 +40,13 @@ invoke_maven() {
     done
   fi
 
+  if [ "$build_scan_publishing_mode" == "off" ]; then
```

Comment:
I would swap the condition such that it checks for `on` instead of `off` (like we do in all other places).

Reply:
Great catch! Addressed in f7f4f8dce143dc2f01536551db1d812b44d1d42a